crates/server/src/admin/handlers/add_client_key.rs

```diff
@@ -94,6 +106,48 @@ pub async fn add_client_key_handler(
         })
 }
 
+//* Register client key to authenticate client requests  */
+pub async fn generate_jwt_token_handler(
+    Extension(state): Extension<Arc<AdminState>>,
+    Json(req): Json<JwtTokenRequest>,
+) -> impl IntoResponse {
+    match generate_jwt_tokens(req, state.store.clone()) {
+        Ok(jwt_tokens) => {
+            let tokens = JwtTokens {
```

Comment:
Can you add
```
#[derive(Debug, Deserialize, Serialize)]
#[serde(rename_all = "camelCase")]
struct JwtTokens {
access_token: String
refresh_token: String
}
```

To match return style that we have on other api endpoints. 
This is what i get on frontend and I would like it to be acessToken, refreshToken
![Screenshot 2024-08-29 at 11 10 45](https://github.com/user-attachments/assets/86a57279-6ed2-4607-804d-fbe4a4f17ee4)

+do the same thing for refresh token

Reply:
Every response should return either `data` or `error`. We have it like that in most endpoints except the first ones.